### PR TITLE
Minor comment and panic message corrections

### PR DESCRIPTION
**Pull request by Mingling94**
Also added a make file to go get dependencies

### CODE REVIEW

**Comment by agundy** [comment]
Also working on overhauling the front end right now. The js changes are unneeded.

**Comment by lukechampine** [comment]
@agundy: if you comment on the "Files changed" tab instead of the individual commit, it will show context in the PR, which is helpful.

**Comment by agundy** [comment]
@lukechampine i was curious why those weren't showing up inline

File: routes.go [hunk 1]
```diff
@@ -5,8 +5,7 @@ import (
 	"net/http"
 )
 
-// Route is an object that defines how a url is called and what function
-// handles the response.
+// Each Route object defines how a url is called and its handler function
```
Comment (agundy) [inline routes.go:8]: This is a superficial change that breaks godoc. The comment string on a function is supposed to start with the name a function.

File: sunfish.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ func NewSunfish() *Sunfish {
 	var err error
 	sf.DBSession, err = mgo.Dial("localhost")
 	if err != nil {
-		fmt.Println("Could not reach a Mongo server. Make you have Mongo configured correctly.")
+		fmt.Println("Could not reach a Mongo server. Ensure Mongo is configured correctly.")
```
Comment (agundy) [inline sunfish.go:28]: Errors are handle differently with one of the outstanding pr's i have. This won't be mergeable

Comment (Mingling94) [inline sunfish.go:28]: Word, these were mostly suggestions :P

File: routes.go (hunk 1 above)
Comment (Mingling94) [inline routes.go:8]: Gotcha, didn't know that's how it worked :P

**Comment by Mingling94** [comment]
@agundy, aight scrapping this PR then if you're overhauling the JS as that was my main concern. Though, get a makefile to install (and keep track of) dependencies if you could

**Pull request closed without merging**